VERSION

```diff
@@ -0,0 +1 @@
+3.1.0-SNAPHOT
```

Comment:
has `3.0.0` version been released already?

Reply:
Not yet. I'm going to do it as soon as webhooks will be set.
Do you think it makes sense to leave 3.0.0-SNAPSHOT now?

Reply:
yeah, IMO ` 3.0.0-SNAPSHOT` is a better choice